docs/guide-contentful.md

```diff
@@ -0,0 +1,110 @@
+# Gridsome Contentful Guide
+
+## Create a Gridsome project
+
+1. `gridsome create my-gridsome-site` to create a new project
+2. `cd my-gridsome-site` to open folder
+3. `gridsome develop` to start local development server
+
+## Create Contentful Account
+
+First you need to create an account with Contentful and select an example of blog space.
+
+Also need API keys (Space ID, Content Delivery API, Content Preview API and Environments - the default should be `master`)
+
+## Installing dependencies
+
+First, install a plugin to link our site and Contentful:
+
+- `yarn add @gridsome/source-contentful`
+
+Then we need to install a markdown processor in our project, since Contentful serves us content in markdown format:
+
+- `yarn add markdown-it`
+
+## Configuration
+
+To begin with, at the root of the project, we will create a `.env` file and fill it with the contents:
+
+```js
+CONTENTFUL_SPACE = "<YOUR_CONTENTFUL_SPACE>";
+CONTENTFUL_TOKEN = "<YOUR_CONTENTFUL_TOKEN>";
+CONTENTFUL_ENVIRONMENT = "<YOUR_CONTENTFUL_ENVIRONMENT>";
```

Comment:
I was running into an error when trying to run `gridsome develop`. It seems to have been the `;`'s in the .env file?